### PR TITLE
[twilio-video] Adds priority setting for Local Track Publications

### PR DESCRIPTION
**Pull request by djungowski**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: See [https://media.twiliocdn.com/sdk/js/video/releases/2.0.0/docs/global.html#MediaStreamTrackPublishOptions]() and [https://media.twiliocdn.com/sdk/js/video/releases/2.0.0/docs/LocalParticipant.html#publishTrack__anchor]()
- [X] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)

### CODE REVIEW

**Comment by djungowski** [comment]
ping

File: types/twilio-video/index.d.ts [hunk 1]
```diff
@@ -583,6 +584,9 @@ export interface LogLevels {
     signaling: LogLevel;
     webrtc: LogLevel;
 }
+export interface MediaStreamTrackPublishOptions extends LocalTrackOptions {
+    priority: Track.Priority;
```
Comment (peterblazejewicz) [inline types/twilio-video/index.d.ts:588]:
```suggestion
    priority?: Track.Priority;
```
should be probably optional, as it defaults to `standard`, am I correct?
https://media.twiliocdn.com/sdk/js/video/releases/2.0.0/docs/global.html#MediaStreamTrackPublishOptions

**Review by peterblazejewicz — COMMENTED** [review]
@djungowski looks OK, this little detail with [optional] property in options to be settled

File: types/twilio-video/index.d.ts (hunk 1 above)
Comment (djungowski) [reply types/twilio-video/index.d.ts]: You are totally right, fixed in bf4abac7e9af18db682a327883398c558037bd18

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@djungowski thx!

**Comment by djungowski** [comment]
Ready to merge

**Pull request merged**